lib/application/view-models/home/update_collection_vm.dart

```diff
@@ -27,32 +37,40 @@ abstract class UpdateCollectionVM extends StateNotifier<UpdateCollectionState> {
 }
 
 class UpdateCollectionVMImpl extends UpdateCollectionVM {
-  UpdateCollectionVMImpl({String? collectionId}) : super(collectionId: collectionId, state: UpdateCollectionLoading()) {
-    loadInitialContent();
+  UpdateCollectionVMImpl({
+    String? collectionId,
+  }) : super(collectionId: collectionId, state: UpdateCollectionLoading()) {
+    loadContent();
   }
 
   @override
   bool get isEditing => collectionId != null;
 
+  UpdateCollectionLoaded get loadedState => state as UpdateCollectionLoaded;
+
   @override
-  Future<void> loadInitialContent() async {
+  Future<void> loadContent() async {
     state = UpdateCollectionLoading();
 
     try {
       // TODO: Call services to load [collectionId] if necessary or create an empty collection metadata
       await Future<void>.delayed(const Duration(seconds: 2));
 
-      // state = UpdateCollectionLoaded(collectionMetadata: CollectionMetadata.empty());
-      state = UpdateCollectionFailedSaving(
-        UrlException.failedToOpen(),
-        metadata: CollectionMetadata.empty(),
-        memosMetadata: const [],
-      );
+      state = UpdateCollectionLoaded(collectionMetadata: CollectionMetadata.empty(), memosMetadata: const []);
+
+      // state = UpdateCollectionFailedSaving(
+      //   UrlException.failedToOpen(),
+      //   metadata: CollectionMetadata.empty(),
+      //   memosMetadata: const [],
+      // );
```

Comment:
I removed, but as I suggested in previous PRs I would ignore this state changes because I normally let them commented to make easy to test the changes. They will be addressed when connecting with the services stuff.

Since this entire branch will land on `main` only after the services are good to go, this "dirty" code will not be merged in `main`.